ui.py

```diff
@@ -298,7 +298,10 @@ def execute(self, context):
 
         # change the menu title to the most recently chosen option
         preset_class = VIEW3D_MT_retarget_presets
-        preset_class.bl_label = bpy.path.display_name(basename(filepath), title_case=False)
+        if bpy.app.version < (2, 93, 0):
```

Comment:
users in older versions will see their preset names in title case. otherwise - exception raises;